modules/swagger-codegen/src/main/resources/qt5cpp/api-header.mustache

```diff
@@ -9,7 +9,9 @@
 
 #include <QObject>
 
-namespace Swagger {
+{{#cppNamespaceDeclarations}}
+namespace {{this}} {
+{{/cppNamespaceDeclarations}}
```

Comment:
@sabras75 Given that the original name spec is Swagger, do we still need the {{#cappNamespaceDeclarations}} .. {{/cppNamespaceDeclarations}} block? I want to keep `namespace Swagger` in the output code if no c++ namespace is provided to make the change backward compatible.